advocacy_docs/supported-open-source/patroni/01-supported_topology.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Supported Topology'
+description: 'EDB supported topology'
+tags:
+  - topology
+  - Patroni
+---
+
+### Description
+
+Main HA components:
+* Postgres cluster: the database cluster, usually consisting of a primary and two or more replicas
+* Patroni: used as the failover management utility
+* etcd: used as a distributed configuration store (DCS), containing cluster information such as configuration, health, and current status
+
+![Supported topology](images/supported-arch.png)
+
+#### Postgres
+
+EDB currently only supports using Patroni and Postgres _physical Streaming Replication_ with a primary server and two or more replicas (replicating directly from the primary).
+
+#### Patroni
+
+Patroni is installed on the Postgres nodes. EDB supports the [latest release](https://patroni.readthedocs.io/en/latest/releases.html#releases).
```

Comment:
@kelpoole @tomjohnkinedb I think we should state the release we are supporting. Today it's 2.1.4. We could also say we support 2.1.x and suggest using the latest point release. 
By looking at what was included in the release notes for the last 4 versions, it's hard to say if that is or isn't the most viable option.
Open to opinions

Reply:
AFAICT, pgdg repositories only contain 2.1.3 and 2.1.4 atm. So latest and latest-1.

In PG, there's an interest in supporting multiple major versions but not in parallel software. Usually, when a bug on an older version is reported, the natural investigation process is looking at the release note to see if already been fixed and trying to reproduce it on the latest release. Then, if a bug fix is needed, it won't be backported to let's say 1.6.6 but fixed in 2.1.5.

So supporting the latest release seems reasonable to me...